packages/terra-markdown/src/Markdown.scss

```diff
@@ -5,3 +5,7 @@
     background-color: #ebebef;
   }
 }
+
+.markdown-body blockquote {
+  color: #24292e;
+}
```

Comment:
Can we add an webdriver.io accessibility test for this change? https://github.com/cerner/terra-core/pull/2209#discussion_r253877135

Reply:
Yeah, it would be good to get some webdrivoer.io tests in place for this component.

Reply:
I'll get on that tomorrow 👍

Reply:
@AlexBrizi any progress?

Reply:
Have been working on it all day and some on Monday. Should have something to show Brett for tomorrow :)